integrations/serverpod_cloud_storage_s3/lib/src/aws_s3_upload/aws_s3_upload.dart

```diff
@@ -156,36 +156,25 @@ class AwsS3Uploader {
 
     /// The filename to upload as. If null, defaults to the given file's current filename.
     required String uploadDst,
+    Duration expires = const Duration(minutes: 10),
+    int maxFileSize = 10 * 1024 * 1024,
     bool public = true,
   }) async {
     final endpoint = 'https://$bucket.s3-$region.amazonaws.com';
-    // final uploadDest = '$destDir/${filename ?? path.basename(file.path)}';
-
-    // final stream = http.ByteStream.fromBytes(data.buffer.asUint8List());
-
-    // final stream = http.ByteStream(Stream.castFrom(file.openRead()));
-    // final length = data.lengthInBytes;
-
-    // final uri = Uri.parse(endpoint);
-    // final req = http.MultipartRequest("POST", uri);
-    // final multipartFile = http.MultipartFile('file', stream, length, filename: path.basename(uploadDst));
 
     final policy = Policy.fromS3PresignedPost(
-        uploadDst, bucket, accessKey, 15, 10 * 1024 * 1024,
-        region: region, public: public);
+      uploadDst,
+      bucket,
+      accessKey,
+      expires.inMinutes,
```

Comment:
Question: If I understand this correctly, the input value here was 15 before, but now we use a default value of 10 minutes. Is this an intentional change?

Reply:
It is intentional as this is the default in the public interface. As a user I would expect that the default is 10 minutes and changing that to 15 will modify the previous default value.

Reply:
Aah, thanks for the explanation! 👍 
Very strange that we just ignored the value before.